### DISCUSSION TITLE
Why email verification shows dangerous sign and landing in spam folder?

### DISCUSSION THREAD

**Discussion by dntandan** [thedevs-network/kutt#462] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32794634/114266305-4190d900-9a13-11eb-85ec-21c05f7424b5.png)

**Comment by poeti8** [comment, marked as answer]
Do you still have this problem? Kutt was blacklsited on spamhaus but it's been removed from the blacklist now.

**Reply by dntandan** [reply]
No,  the verification email is now landing in the primary inbox. This issue has been resolved now. 

Thanks for getting back with the answer.
